### DISCUSSION TITLE
rofi not loading with the correct font

### DISCUSSION THREAD

**Discussion by ree-see** [davatorium/rofi#1980] [(Possible) Issues]
So this is my first github discussion post, I don't know if there is a standard with posting an issue, but I just started configuring rofi to use JetBrains Mono. When I go to run rofi it doesn't use JetBrains it uses what I think is some type times new roman. Sorry if this wasn't the greatest issue discussion post.

**Comment by DaveDavenport** [comment]
Can you explain how you 'configure' ..  
f.e. post your config file, or the commandline (if you pass it this way) you use?

**Reply by ree-see** [reply]
`~/.config/rofi/themes/config.rasi
configuration = {
     ...
     font: "JetBrains Mono 10";
}`

I tried "JetBrainsMono 10", and even the path to the ttf-jetbrains-mono I knew that wasn't going to work but you never know.

**Reply by DaveDavenport** [reply]
> ~/.config/rofi/themes/config.rasi configuration = { ... font: "JetBrains Mono 10"; }

What is this?  it does not look like a valid config. this "abstract form" makes it impossible for me to check if it is correct and at right location....

Some notes:
  * the config file path looks wrong. (should not be in subdir themes) https://davatorium.github.io/rofi/current/rofi-theme.5/#getting-started-with-theming
  * '=' is not allowed at that point.
  * configuration is the wrong block, as its a theme setting.

**Reply by DaveDavenport** [reply]
This for me works.

```css
* {
   font: "JetBrains Mono 10";
}
```
